lib/calrom/option_parser.rb

```diff
@@ -252,5 +260,15 @@ def build_date_range(range_type, year, month, day, another_day=nil)
 
       range
     end
+
+    def last_month
+      Date.new(Time.now.year, (Time.now.month - 1), 1)
+    end
+
+    def next_month
+      last_day_next_month = (Time.new(Time.now.year, ((Time.now.month + 1) % 12) + 1) - 1).day
```

Comment:
Unhandled edge case: this won't work correctly if current month is December.

Reply:
Should I use your method `next_month_beginning`?

Reply:
No. `Month#next_month_beginning` computes next month's beginning, you need next month's end.

(Moreover, it's poorly written and could be significantly simplified. At the time of writing it I was unaware of `Date#next_month` and other useful [`Date`](https://ruby-doc.org/stdlib-2.6.5/libdoc/date/rdoc/Date.html) methods.)